wordpress-mobile/AztecEditor-iOS
# Removes a lot of old code.

Removes a lot of old code that should be cleaned up as part of #492.

Unfortunately this PR doesn't fix all of the unit tests, since some of them are genuinely broken (the remaining ones).

**How to test:**

Run the unit tests, evaluate that the broken ones are genuinely broken and not part of old code.

Use the editor lightly and make sure nothing new is broken when compared to the main branch.
